src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/magical/staves/StormStaff.java

```diff
@@ -33,15 +33,15 @@
  */
 public class StormStaff extends LimitedUseItem {
 
-    public static final int MAX_USES = 8;
+    public static final int STORM_MAX_USES = 8;
```

Comment:
why change this?

Reply:
i added max uses to fire aswell
so thats why i made that

Reply:
you dont have to its inside the class

Reply:
![image](https://user-images.githubusercontent.com/44972470/141357166-3734aeed-5ba1-4743-9f1d-b23316c2c7b3.png)
you are calling it here anyways from the class so it doesnt matter what the name is